bin/cli.js

```diff
@@ -40,7 +40,7 @@ async function runServer(file) {
       console.error(code);
       throw TypeError(`Cannot find Invokable function 'handle' in ${code}`);
     }
-    ON_DEATH(_ => {
+    nodeCleanup(_ => {
       server.close();
```

Comment:
I think the only thing that needs to be added is a `process.exit(0)` after the `server.close` instead of changing the module.

Reply:
@helio-frota can you give this a try in your test scenario?

Reply:
@lholmquist @lance that works thanks!